wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -215,6 +216,7 @@ namespace :wagn do
           end
         end
         Card::Cache.reset_global
+        # FIXME: can this be associated with the machine module somehow?
```

Comment:
It might be helpful to split out tasks that are just for core developers.  We could even put them in the repo root (as in, not in the gems), because this is at best noise for non-core-developers, and at worst it could be really confusing.

`bootstrap:clean` and `bootstrap:dump`